edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/SystemModules.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Core
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    public class SystemModules : IEquatable<SystemModules>
+    {
+        [JsonConstructor]
+        public SystemModules(IEdgeAgentModule edgeAgent, IEdgeHubModule edgeHub)
```

Comment:
Why is this new class needed?

Reply:
Not a new class, just moved it to its own file since it was getting large. It was earlier in DeploymentConfig.cs